package.json

```diff
@@ -25,16 +25,18 @@
     "@changesets/cli": "^2.16.0",
     "@github/prettier-config": "0.0.4",
     "@primer/primitives": "^5.1.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.0.1",
     "jest": "^27.0.6"
   },
   "peerDependencies": {
-    "eslint": ">=4.19.0",
+    "eslint": "^8.0.1",
```

Comment:
I ran into dependency conflict because of the eslint version specified in `eslint-plugin-github` so I had to bump this up.

Reply:
not sure why it broke a test